doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -106,6 +106,15 @@ Matter
 * Enabled :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option by default, including for builds without factory data support.
   The firmware now erases all flash pages in the non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
 
+* RAM usage optimizations:
+
+  * Reduced number of network and Matter stack buffers and packets.
+  * Disabled SSL server support.
+  * Reduced the Main, Chip, mbedTLS, and OpenThread stack sizes according to the measurements.
```

Comment:
```suggestion
  * Reduced the stack sizes for Main, Matter and OpenThread threads.
```
"according to the measurements" applies to all optimizations, so maybe this should be added before or after the bullet list?
Also, for mbedTLS we reduced the heap, not stack.